### ISSUE TITLE
[Bug]: Unable to Remove User

### ISSUE THREAD

**Issue by astupidmoose** [btcpayserver/btcpayserver#4641]
### What is your BTCPay version?

BTCPay Server v1.7.5

### How did you deploy BTCPay Server?

Docker Install

### What happened?

I was unable to remove, disable or demote an admin user. 

### How did you encounter this bug?

*For the purpose of this bug report I have renamed the user name to "A" as opposed to the key generated name. 

Secondary admin user forgot password, so they attempted a reset. The Reset failed as SMTP wasn't setup and the user was locked out:

2023-02-14 18:04:31.160 +00:00 [WRN] Should have sent email, but email settings are not configured
2023-02-14 18:05:40.984 +00:00 [WRN] User 'A' account locked out.

I logged in from another admin user and attempted to delete the account. 

2023-02-14 18:10:54.484 +00:00 [ERR] Failed to delete user A
2023-02-14 18:11:05.757 +00:00 [ERR] Failed to delete user A

I then tried to remove admin status, which also errored. 

2023-02-14 18:11:27.445 +00:00 [ERR] Error setting admin status for user A

I think tried to disable the user:

2023-02-14 18:11:38.327 +00:00 [ERR] Failed to set lockout for user A

Followed by further attempts to delete:
2023-02-14 18:12:37.518 +00:00 [ERR] Failed to delete user A
2023-02-14 18:15:37.999 +00:00 [ERR] Failed to delete user A

Finally I gave up, setup SMTP and the user was able to reset their password and login. 

2023-02-14 18:22:52.218 +00:00 [INF] User 'A' logged in.

### Relevant log output

```shell
2023-02-14 18:04:31.160 +00:00 [WRN] Should have sent email, but email settings are not configured
2023-02-14 18:05:40.984 +00:00 [WRN] User 'A' account locked out.
2023-02-14 18:10:54.484 +00:00 [ERR] Failed to delete user A
2023-02-14 18:11:05.757 +00:00 [ERR] Failed to delete user A
2023-02-14 18:11:27.445 +00:00 [ERR] Error setting admin status for user A
2023-02-14 18:11:38.327 +00:00 [ERR] Failed to set lockout for user A
2023-02-14 18:12:37.518 +00:00 [ERR] Failed to delete user A
2023-02-14 18:15:37.999 +00:00 [ERR] Failed to delete user A
2023-02-14 18:22:52.218 +00:00 [INF] User 'A' logged in.
```

### What browser do you use?

Chrome

### Are you sure this is a bug report?

- [X] I confirm this is a bug report

**Comment by NicolasDorier** [comment]
I know you have manual install right?
Are you using sqlite backend?

**Comment by NicolasDorier** [comment]
Also have weird window

![image](https://user-images.githubusercontent.com/3020646/218925256-003b7abe-6b1c-4582-ac0f-4723435af177.png)

ping @dennisreimann

**Comment by NicolasDorier** [comment]
I can repro the problem, a restart fix it. Trying to find out what is going on.

**Comment by dennisreimann** [comment]
Fixed the UI in 9aff143d40283f4b8b8396bc52a7c5bec26fc740
